ambry-server/src/test/java/com.github.ambry.server/AmbryRequestsTest.java

```diff
@@ -149,6 +152,17 @@ public AmbryRequestsTest() throws IOException, ReplicationException, StoreExcept
     ambryRequests = new AmbryRequests(storageManager, requestResponseChannel, clusterMap, dataNodeId,
         clusterMap.getMetricRegistry(), FIND_TOKEN_FACTORY, null, replicationManager, null, false,
         storeKeyConverterFactory);
+    storageManager.start();
+  }
+
+  /**
+   * Close the storageManager created.
+   */
+  @After
+  public void after() throws InterruptedException {
+    if (storageManager != null) {
```

Comment:
If `storageManager` gets a value in constructor, then we don't need to check `null`.